brownie/project/main.py

```diff
@@ -835,7 +835,7 @@ def _maybe_retrieve_github_auth() -> Dict[str, str]:
 
 def _install_from_github(package_id: str) -> str:
     try:
-        path, version = package_id.split("@")
+        path, version = package_id.split("@", 1)
         org, repo = path.split("/")
```

Comment:
maybe this needs a similar fix?

Reply:
Github does not allow '/' in org and repo name.